src/it/java/org/checkstyle/suppressionxpathfilter/AbstractXpathTestSupport.java

```diff
@@ -210,8 +237,7 @@ protected void runVerifications(DefaultConfiguration moduleConfig,
         treeWalkerConfigWithXpath.addChild(createSuppressionXpathFilter(moduleConfig.getName(),
                 generatedXpathQueries));
 
-        final Integer[] warnList = getLinesWithWarn(fileToProcess.getPath());
-        verify(moduleConfig, fileToProcess.getPath(), expectedViolations, warnList);
+        verify(moduleConfig, fileToProcess.getPath(), expectedViolations);
```

Comment:
It use to fail if warn trailing comment is not placed in input file.
Is it going to continue to work ?

Reply:
No, this is because normal test doesn't have this feature except in BDD, and xpath can't currently be swapped for bdd.

Reply:
@romani Is this an issue?

Reply:
@romani ping

Reply:
I am confused that we still put such "warn" in Input files
https://github.com/checkstyle/checkstyle/blob/b6301c189ef5d2d731a755fa9a2e84aa0677494a/src/it/resources/org/checkstyle/suppressionxpathfilter/annotationusestyle/SuppressionXpathRegressionAnnotationUseStyleEight.java#L3

are they redundant after this commit ?
them use to be a marker to show line for reviewer to understand what is target of xpath. Reading of xpath is not pleasant, so extra hint is good help.